api/src/uvisor-input.S

```diff
@@ -27,6 +28,12 @@
     .thumb
     .thumb_func
 
+/* The uvisor_privcall table comes 0x100 bytes after uvisor_init. If
+ * uvisor_init ever becomes too large that 0x100 bytes isn't enough, a linker
+ * error will be generated when the linker script tries to move "." backwards,
+ * making uvisor_privcall overlap with main_entry. */
+.set uvisor_privcall, uvisor_init + 0x100
```

Comment:
Could you add a `FIXME` here?

In particular, I think that it would be nice in the future to expose this table in the same way we expose the symbol `uvisor_init`.

`uvisor_init` is a unique symbol that represents the whole uVisor binary blob, and it is used as *code*.

Similarly, a symbol `uvisor_data` could be used to export the table with all function pointers and other data that uVisor wants to share with the host OS.